public/locales/en/common.json

```diff
@@ -373,8 +373,19 @@
       "upcomingDistributionDate": "Upcoming distribution date",
       "upcomingDistributionValue": "Upcoming distribution value",
       "usdValue": "USD Value",
-      "weeklyDillAmount": "Weekly DILL 🔒",
-      "weeklyPickleAmount": "Weekly PICKLEs distributed"
```

Comment:
@bcstryker why was this removed? It breaks the dill chart labels/tooltips.

Reply:
wasn't supposed to be, my bad, will re-add if you haven't already

Reply:
weird, on checking master, this change is not present

Reply:
replaced and merged, my bad 😬

Reply:
Awesome, thank you!

Reply:
closing this